stumpy/aampdist_snippets.py

```diff
@@ -231,11 +257,14 @@ def aampdist_snippets(
         mask = snippets_profiles[i] <= total_min
         snippets_fractions[i] = np.sum(mask) / total_min.shape[0]
         total_min = total_min - mask.astype(np.float64)
+        slices = _get_mask_slices(i, mask)
+        snippets_regimes = np.append(snippets_regimes, slices, axis=0)
```

Comment:
Assuming you start with an empty Python list (see above) then you'd do `snippets_regimes.append(slices)` here 

and then, afterward, you'd convert this list to an array:

```
snippets_regimes = np.array(snippets_regimes)
```